app/javascript/components/journey/SolutionFilter.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { usePanel } from '../../hooks/use-panel'
+import { GraphicalIcon } from '../common'
+
+type SolutionFilterOption = {
+  value: string
+  title: string
+}
+
+const OPTIONS: SolutionFilterOption[] = [
```

Comment:
I dont know what the real filter options are.

Reply:
Me neither!!